ElectronClient/app/gui/NotePropertiesDialog.jsx

```diff
@@ -347,7 +363,17 @@ class NotePropertiesDialog extends React.Component {
 		const formNote = this.state.formNote;
 
 		const buttonComps = [];
-		buttonComps.push(<button key="ok" style={styles.button} onClick={this.okButton_click}>{_('Apply')}</button>);
+		buttonComps.push(
+			<button
+				key="ok"
+				style={styles.button}
+				onClick={this.okButton_click}
+				ref="okButton"
+				onKeyDown={(event) => this.onKeyDown(event)}
```

Comment:
You can simply do `onKeyDown={this.onKeyDown}` since the handler is already bound to this component, and that will avoid an anonymous function.